.editorconfig

```
@@ -18,3 +18,7 @@ indent_size = 2
 [coderd/database/dump.sql]
 indent_style = space
 indent_size = 4
+
+[coderd/database/queries/*.sql]
+indent_style = tab
+indent_size = 4
```

```suggestion
indent_size = 4

```

classic missing trailing newline